index.js

```diff
@@ -274,10 +281,10 @@
     //
     // Returns an Object {n, s, e, w, nw, sw, ne, se}
     function getScreenBBox() {
-      var targetel   = target || d3.event.target;
+      var targetel   = target || d3.event.target
 
-      while ('undefined' === typeof targetel.getScreenCTM && 'undefined' === targetel.parentNode) {
-          targetel = targetel.parentNode;
+      while (targetel.getScreenCTM == null && targetel.parentNode == null) {
```

Comment:
The `== null` comparison allows for a convenient two-for-one check if a value is either `null` or `undefined` (since `undefined == null`), because except in rare scenarios you usually want to treat both values the same way. As an added bonus it's also shorter than a `typeof` check.  It's the only use of `==` over `===` I tend to use ([`eqeqeq`](http://eslint.org/docs/rules/eqeqeq) FTW).

(Looks better it does, no longer [`yoda`](http://eslint.org/docs/rules/yoda) it is 😄 )